meta-ibm/recipes-phosphor/chassis/mux-workarounds/mux-workarounds.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+# everest system only
+if [ $(fw_printenv fitconfig | grep everest) ]; then
```

Comment:
I'm not sure, I guess it would be in the VPD but I think this should run before VPD collection gets going... guess I could test afterwards, but still not sure how to check the right VPD?

Reply:
I take it there's no adverse effects if we run it against hardware that doesn't need it? Waiting for vpd collection is probably going to add more complexity then we want.